### PR TITLE
add synonyms to epilepsy terms

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
address #8135

### CODE REVIEW

**Comment by twhetzel** [comment]
Does this still need to be reviewed? The [linked issue](https://github.com/monarch-initiative/mondo/issues/8135) is closed now.

**Comment by nicolevasilevsky** [comment]
yes please, thanks @twhetzel!

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -142304,6 +142304,7 @@ synonym: "Angelman syndrome" EXACT CLINGEN_LABEL [MONDO:Lexical, OMIM:105830]
 synonym: "Angelman syndrome (Type 1)" NARROW [DECIPHER:4]
 synonym: "Angelman syndrome (Type 2)" NARROW [DECIPHER:54]
 synonym: "Angelman syndrome chromosome region" RELATED [OMIM:105830]
+synonym: "Angelman’s Syndrome" EXACT [PMID:37498137]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:142307]: syndrome should not be capitalized

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -355678,6 +355683,7 @@ synonym: "chromosome 20 ring" RELATED [GARD:0001334]
 synonym: "R20" RELATED ABBREVIATION [GARD:0001334]
 synonym: "Ring 20" RELATED [GARD:0001334]
 synonym: "Ring chromosome 20 syndrome" RELATED [GARD:0001334]
+synonym: "ring chromosome 20 syndrome" EXACT [PMID:37498137]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:355686]: @nicolevasilevsky the other "ring chromosome 20 syndrome" x-refs are related. Could you please update according to whether the synonym should be exact or related? Thank you!

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -535490,10 +535514,13 @@ subset: clingen {source="MONDO:CLINGEN"}
 subset: gard_rare {source="GARD:10430", source="MONDO:GARD"}
 subset: nord_rare {source="NORD:1061", source="MONDO:NORD"}
 subset: rare
+synonym: "Dravet" EXACT [PMID:37498137]
 synonym: "Dravet syndrome" EXACT CLINGEN_LABEL [OMIM:607208]
 synonym: "DS" EXACT ABBREVIATION [Orphanet:33069]
 synonym: "myoclonic epilepsy, severe, of infancy" RELATED [GARD:0010430]
+synonym: "severe myoclonic epilepsy of infancy" EXACT [PMID:37498137]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:535521]: There are 2 synonyms that look the same, one is "related" and the other one is "exact". Could you please review and update accordingly? Thank you

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -535490,10 +535514,13 @@ subset: clingen {source="MONDO:CLINGEN"}
...
 synonym: "SME" RELATED ABBREVIATION [GARD:0010430]
+synonym: "SMEB" EXACT [PMID:37498137]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:535523]: same as above regarding SME and SMEB

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky There are a few synonyms that are the same (or very similar) and are both exact and related. Please review these and keep whichever is the correct one. 
(Note that I think we brought all GARD synonyms as "related", so I suspect the synonyms from GARD are the one with the incorrect type)

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:355686]: fixed

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:142307]: fixed

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:535521]: updated

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:535523]: fixed

**Comment by nicolevasilevsky** [comment]
@sabrinatoro I cleaned this up, it is ready for your re-review, thanks!

**Review by sabrinatoro — APPROVED** [review]
Thank you @nicolevasilevsky! 
Approving and merging after QCs have passed.

**Pull request merged**